Components/Buttons/button.css

```diff
@@ -0,0 +1,123 @@
+.button{
+    font-family: poppins;
+    width: fit-content;
+    min-width: 150px;
+    height: fit-content;
+    font-size: 1rem;
+    padding: 0.5rem 1rem;
+    border-radius: 0.5rem;
+    cursor: pointer;
+    background-color: transparent;
+}
+.link{
+    color: var(--primary-color);
+    font-size: 1rem;
+    padding: 0rem;
+    background-color: transparent;
+    cursor: pointer;
+}
+.link:hover{
+    text-decoration: underline;
+}
+.button-primary{
+    background-color: var(--primary-color);
+    color: white;
+    border: 2px solid var(--primary-color);
+}
+.button-primary:hover{
+    background-color: var(--primary-color-dark);
+}
+.button-secondary{
+    background-color: var(--secondary-color);
+    color: white;
+    border: 2px solid var(--secondary-color);
+}
+.button-secondary:hover{
+    background-color: var(--secondary-color-dark);
+}
+.button-success{
+    background-color: var(--success-button);
+    color: white;
+    border: 2px solid var(--success-button);
+}
+.button-success:hover{
+    background-color: var(--dark-green);
```

Comment:
The success button color on hover looks great but it could be a more subtle version as I find that the color change might be jarring when hovered on the button.

Reply:
Sure i would do it